server/matchmaker/matchmaker_queue.py

```diff
@@ -48,9 +48,12 @@ def __init__(
         self._is_running = True
 
         self.timer = PopTimer(self.queue_name)
-        asyncio.ensure_future(self.queue_pop_timer())
         self._logger.debug("MatchmakerQueue initialized for %s", queue_name)
 
+    async def initialize(self):
+        loop = asyncio.get_running_loop()
+        loop.create_task(self.queue_pop_timer())
```

Comment:
```suggestion
        asyncio.create_task(self.queue_pop_timer())
```

Reply:
Then you probably also don't want the `loop` from the previous line?